packages/wix-ui-tpa/src/components/Dropdown/DropdownError.tsx

```diff
@@ -5,17 +5,19 @@ import { Tooltip } from '../Tooltip';
 import { TPAComponentProps } from '../../types';
 import { DATA_HOOKS, ICON_SIZE } from './constants';
 import { st, classes } from './DropdownError.st.css';
+import { Placement } from 'wix-ui-core/popover';
 
 interface DropdownErrorProps extends TPAComponentProps {
   errorMessage: string;
+  placement?: Placement;
 }
 
 export const DropdownError: React.FC<DropdownErrorProps> = (props) => {
   return (
     <Tooltip
       className={st(classes.root, props.className)}
       data-hook={DATA_HOOKS.errorTooltip}
-      placement="top-end"
+      placement={props.placement || 'top-end'}
```

Comment:
I would move the default prop value to this file, and then you could just write
```suggestion
      placement={props.placement}
```

Reply:
done